lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -158,7 +158,8 @@ class ReportUIFeatures {
       this.addButton({
         text: Util.i18n.strings.viewTreemapLabel,
         icon: 'treemap',
-        onClick: () => ReportUIFeatures.openTreemap(this.json),
+        onClick: () => ReportUIFeatures.openTreemap(
+          this.json, this._dom.isDevTools() ? 'url' : 'postMessage'),
```

Comment:
is there any reason to keep the `postMessage` approach around? It would be a lot simpler to reason about if every client used just the single path (ideally we'd be able to revert some of #11667)

Reply:
> treo is already using it, so we'd break them

I mean they deployed within like 6 hours of release and they get it for free from the report, so seems ok? We can turn off _sending_ via `postMessage` but still support receiving messages for a while to make sure nothing breaks